Scarb.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "openzeppelin"
+version = "0.1.0"
```

Comment:
version number TBD. we should decide whether to restart versioning or to pick up [where we left](https://github.com/OpenZeppelin/cairo-contracts/releases) (probably the latter)

Reply:
I understand 😃 Unfortunately, this field is required, so we need to set one now. https://docs.swmansion.com/scarb/docs/reference/targets#library